src/composables/accounts.ts

```diff
@@ -7,6 +7,7 @@ export function useAccounts({ store }: IDefaultComposableOptions) {
   const accounts = computed((): IAccount[] => store.getters.accounts);
   const account = computed((): IAccount => store.getters.account);
   const accountsAddressList = computed(() => accounts.value.map((acc) => acc.address));
+  const isLoggedIn = computed(() => Object.keys(account.value).length > 0);
```

Comment:
@peronczyk Could you explain this method? How it is determining if the user has logged in or not?

Reply:
This code was just moved from the `getters.js` file - it is not my invention. But actually makes sense as the `account` is always an object, which is empty if there is no user logged in. Or you think it should be updated?

Reply:
Okay.  We may have to revisit this when we add biometric authentication to the superhero wallet.

Reply:
Definitely.